docs/design/design-doc-process.md

```diff
@@ -0,0 +1,64 @@
+# Design Document Process
+
+Design documents will be checked in as Markdown files in `docs/design` folder.
+
+Design documents should have an editor, at least one discussion meeting, and at least two reviewers from the architecture board.
+
+The editor is responsible for:
+* soliciting authors or authoring the document themselves,
+* scheduling discussion meetings,
+* finding reviewers,
+* posting the document in our [forum](https://forum.storj.io/c/engineer-amas/design-draft), and
+* getting the document finalized and merged
+
+The editor is also responsible for making a list of epics and tickets after the design document is finalized and merged.
+
+The reviewers are responsible for reviewing the clarity and reasonableness of the document.
+
+The discussion meeting should have at least three people present. Invitees should familiarize with the document prior to the discussion. The reviewers should be present. If there are open problems after the meeting, then the meeting should be repeated.
+
+One of the reviewers must be an architecture owner, currently:
+
+* Egon (@egonelbre),
+* Kaloyan (@kaloyan-raev),
+* JT (@jtolds),
+* Jens (@littleskunk),
+* unless agreed otherwise
+
+The other reviewer should be someone with significant distributed systems expertise, currently:
+
+* Paul (@thepaul),
+* Simon (@simongui),
+* Jeff (@zeebo),
+* Matt (@brimstone),
+* unless agreed otherwise.
+
+However, it is expected that there should be feedback from the engineering team, DevOps team, data science team, UX team, QA team, and the community.
+
+## Template
+
+The design document uses `TEMPLATE.md` as a guide. However, it can be modified when necessary.
+
+The template has the following sections:
+
+Abstract
+ : which gives an overview of the design document and what it accomplishes. It does not go into details about the problem.
```

Comment:
This formatting does not seem to render well in Markdown. How is it supposed to look like?

Reply:
Roughly as it does in github. But I can try making it better. Some markdown formatters support it as definition lists, but I guess github doesn't.

Reply:
hopefully it's better now

Reply:
what if we do it as a bullet list?

Reply:
Longer paragraphs as bulleted list ends up harder to read and I feel we will clarify and add information to these paragraphs over time.

Reply:
You can see how it looks in the latest PR.